openprescribing/frontend/management/commands/import_measures.py

```diff
@@ -102,22 +107,24 @@ def parse_options(self, options):
         else:
             options['measure_ids'] = [
                 k for k, v in parse_measures().items() if 'skip' not in v]
-        options['months'] = []
-        if 'month' in options and options['month']:
-            options['start_date'] = options['end_date'] = options['month']
-        elif 'month_from_prescribing_filename' in options \
-             and options['month_from_prescribing_filename']:
-            filename = options['month_from_prescribing_filename']
-            date_part = re.findall(r'/(\d{4}_\d{2})/', filename)[0]
-            month = datetime.datetime.strptime(date_part + "_01", "%Y_%m_%d")
-
-            options['start_date'] = options['end_date'] = \
-                month.strftime('%Y-%m-01')
-        else:
-            l = ImportLog.objects.latest_in_category('prescribing')
-            options['start_date'] = "%s-%02d-%02d" % (
-                l.current_at.year - 5, l.current_at.month, l.current_at.day)
-            options['end_date'] = l.current_at.strftime('%Y-%m-%d')
+        if not options['start_date']:
+            if 'month' in options and options['month']:
+                options['start_date'] = options['end_date'] = options['month']
+            elif 'month_from_prescribing_filename' in options \
+                 and options['month_from_prescribing_filename']:
```

Comment:
I can't see us ever using `month_from_prescribing_filename` -- can we drop it?

Reply:
I wondered about this but left it, because I wondered if we'd need it to restore from old files.

But you're probably right, on reflection